Formula/k/kin.rb

```diff
@@ -0,0 +1,43 @@
+class Kin < Formula
+  include Language::Python::Virtualenv
+
+  desc "Sane PBXProj files"
+  homepage "https://github.com/Serchinastico/Kin"
+  url "https://files.pythonhosted.org/packages/fe/36/f83076675f5542af2418d3c75b1f4b5de9b895c37e418a086ce40a05f1bb/kin-2.1.5.tar.gz"
+  sha256 "6ee259cefa1ef7e565562f226c4fe4647990fa957b2c172b633afad3fd41c89d"
+  license "Apache-2.0"
+  head "https://github.com/Serchinastico/Kin.git", branch: "main"
+
+  depends_on "python@3.11"
```

Comment:
any reason why 3.12 not supported?

Reply:
The output of `brew audit kin` was "use the canonical name 'python@3.11'", probably because of this [alias](https://github.com/Homebrew/homebrew-core/blob/442f9cc511ce6dfe75b96b2c83749d90dde914d2/Aliases/python). I will change it to 3.12.